next/components/organisms/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,71 @@
+import { MenuMainItem } from '@bratislava/ui-bratislava'
+import BANavBar from '@bratislava/ui-bratislava/BANavBar/BANavBar'
+import * as Sentry from '@sentry/nextjs'
+import { useRouter } from 'next/router'
+import { useTranslation } from 'next-i18next'
+import * as React from 'react'
+
+import { usePageContext } from '../../layouts/PageContextProvider'
+import CookieConsent from '../CookieConsent'
+import NavBarHeader from './NavBarHeader/NavBarHeader'
+import NavMenu from './NavMenu/NavMenu'
+
+interface IProps {
+  menuItems: MenuMainItem[]
+  pageColor?: string
+}
+
+const NavBar = ({ menuItems, pageColor }: IProps) => {
+  const router = useRouter()
+  const { locale, localizations = [] } = usePageContext()
+  const [t] = useTranslation('common')
+
+  const handleLanguageChange = async ({ key }: { key: string }) => {
+    const path = localizations.find((l) => l.locale === key)?.slug || ''
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    try {
+      await router.push(`/${path}`, undefined, { locale: key })
+    } catch (error) {
+      Sentry.captureException(error)
+    }
+  }
+
+  return (
+    <>
+      <CookieConsent />
```

Comment:
Why here?

Reply:
Just copied from older code, not refactored yet.